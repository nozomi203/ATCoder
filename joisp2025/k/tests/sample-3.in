7
1 2 1 3 3 2
5 2 8 9 4 0 5
10
1 3 1
2 4 10
3 2
1 6 3
1 2 0
3 1
3 4
2 5 6
3 5
3 3
