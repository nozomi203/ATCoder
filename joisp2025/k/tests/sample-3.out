6
18
19
6
0
